<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameGui</class>
 <widget class="QWidget" name="GameGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147,181,207);
border-radius: 10px;</string>
   </property>
   <widget class="QLabel" name="label_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
   color:rgb(134, 134, 134);
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
QLabel:hover{
   background-color:rgb(58,149,222);
   color:white;

}</string>
    </property>
    <property name="text">
     <string>一</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_close">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>0</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
   color:rgb(134, 134, 134);
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
QLabel:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226,225,228);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>玩家</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_uerName">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string>LSX</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>对手</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_opName">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>当前操作者</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_curPlayer">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>当前状态</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_curStatus">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>300</y>
      <width>281</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(226,225,228);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget
{
background: rgb(226,225,228);; border: 0px;
}
QScrollBar:vertical
{
background: rgb(226,225,228);
width:8px;
}
QScrollBar::handle:vertical
{
background: rgb(186,204,217	);
border-radius: 4px;
border: none;
min-height: 20px;
}
QScrollBar::handle:vertical:hover
{
background: rgb(128,128,128);
border: none;
min-height: 20px;
}
QScrollBar::sub-line:vertical{border: none;}
QScrollBar::add-line:vertical{border: none;}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="MyTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>415</width>
       <height>415</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{background-color:rgb(255, 255, 255)}
QTableWidget:hover{rgb(170, 85, 255);}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 16pt &quot;微软雅黑 Light&quot;;
color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>玩家</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_down">
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
        <property name="text">
         <string>向下↓</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_up">
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
        <property name="text">
         <string>向上↑</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_right">
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
        <property name="text">
         <string>向右→</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_left">
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
        <property name="text">
         <string>向左←</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 25 12pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(85, 170, 255);
color:white;
border-radius: 4px;
}</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>108</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 170, 255);
font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
     </property>
     <property name="text">
      <string>单点</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="MyTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>415</width>
       <height>415</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{background-color:rgb(255, 255, 255)}
QTableWidget:hover{rgb(170, 85, 255);}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 16pt &quot;微软雅黑 Light&quot;;
color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>对手</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">mytablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
